src/state-schemas/__tests__/state-schema.test.js

```diff
@@ -44,16 +44,16 @@ describe('cell schemas should all have disjoint properties', () => {
 describe('state schemas should all have disjoint properties except for "cells" and "viewMode', () => {
   it('mirroredStateProperties, editorOnlyStateProperties have correct intersection', () => {
     expect(getPropertyIntersection(mirroredStateProperties, editorOnlyStateProperties))
-      .toEqual(['cells', 'cellClipboard', 'viewMode'])
+      .toEqual(['cellClipboard', 'viewMode'])
```

Comment:
i think we can get rid of cellClipboard in another PR, but maybe stick a FIXME in here as a reminder